### ISSUE TITLE
Nemo file manager crashes and system unresponsive when phone is connected in USB Debugging Mode

### ISSUE THREAD

**Issue by adrummer354** [linuxmint/nemo#3394]
When I have my USB phone connected and in USB Debugging Mode, nemo will often become unresponsive and cause my system to crash. It will hang and prevent any file manager activity in the system. When shutting down the system the system shows "Nemo Unresponsive" messages. 

This has occurred with the same USB phone device across multiple systems and after reinstalling the OS. System info:

https://termbin.com/l27o

I have tried multiple kernels, reinstalling the OS. Other users on reddit reporting this issue is present with phones connected in this mode but it wont occur for them if the phone device gets ejected.

**Comment by Not-Sure-Whats-Going-On** [comment]
I can confirm this problem on updated Mint 20.3, Cinnamon 6.0.4 and 6.5.0-28-generic kernel with a Pixel6 phone when connecting via USB-C.

**Comment by olisnr** [comment]
i have the same problem. 
but its only with `nemo` `nautilus` works perfekt. may be the nemo people could make a diff to nautilus?

**Comment by olibac** [comment]
I confirm too. I am using LMDE6 and USB debugging was not enabled.

**Comment by joegiglio** [comment]
Same issue.  Pixel 8a, Linux Mint 22.1 Cinnamon.  

I plugged in the phone to charge and all was well.  Nemo became unresponsive, so I killed it from System Monitor.  If I tried again, the process would open but no GUI would appear.  Once I unplugged the phone, all was well. 

Got me freaked out for a minute. :)

**Comment by Not-Sure-Whats-Going-On** [comment]
This is still happening to me on Mint 22.1, Nemo 6.4.3 when Pixel 6 sleeps while connected via USB.

**Comment by olisnr** [comment]
@Not-Sure-Whats-Going-On will not be solved i think... mint-develtoper have iphones (or even mint-phones (using FOSH riscVs)) :)

nautilus is working

**Comment by jjberk** [comment]
I wrote this in the Linux Mint forum:

I had the problem of Nemo sometimes hanging/being unresponsive for a long time ( > 10 minutes). In the end, it seems that this is related to an attached smartphone to an USB port. As soon as I remove the smartphone or eject it, Nemo is immediately fine again. Specifics: the smartphone is set to "charge only". As soon as it is attached with "File sharing" no problem occurs.

**Comment by martindholmes** [comment]
I have this too on Ubuntu 24.04 with Nemo as the default file manager. If I lock the screen with the phone connected, the desktop seems to die and I'm unable to log back in.

**Comment by olisnr** [comment]
we should trow this apple shit away and use linux phones...

**Comment by Jeremy7701** [comment]
Well, it works perfectly well on my old Samsung mobile, also fine on a Kobo ebook.

**Comment by NeverGivinUp** [comment]
I have this problem with Linux Mint 22.1 Cinnamon Version 6.4.8 using a Samsung S24. Using  a Google phone is not a solution.

**Comment by ghost** [comment]
I have this problem on Linux Mint 22.1 with Nemo 6.4.5 with non-Pixel phone

**Comment by martindholmes** [comment]
@olisnr This is not an Apple issue; I've had the same problem with two Pixel phones, and still have it. I can elaborate a bit, though: If I start Nemo, nothing happens, but if I then pull the cable from the phone, Nemo pops up immediately. I think Nemo is doing some kind of interrogation of mounted things, and for some reason can't get the appropriate response from the connected phone, but is unable to stop trying while the phone is connected.

**Comment by olisnr** [comment]
@martindholmes yes somthing stucks. can You debug it?

**Comment by martindholmes** [comment]
@olisnr What do you need me to do?

**Comment by claudiux** [comment]
The phone must be unlocked for this to work properly. Increase its activity time without having to re-authenticate.

**Comment by Robolta** [comment]
Same issue here with a Pixel 4a, Linux Mint 22.2, Cinnamon 6.4.8, and Nemo 6.4.5  It immediately opened right after unplugging the phone.

**Comment by olisnr** [comment]
@martindholmes sorry, i didnt see Your post. i would like You to verify how the nautilus and the nemo filemanager differ in handling the iphone file-system.

something differs. in nautilus i never noticed a blocking of the program. maybe the nautilus-way of handling could be used also in nemo.

**Comment by martindholmes** [comment]
I only connect the phone to charge it; I don't mount the filesystem. Nautilus doesn't have the problem at all.

**Comment by olisnr** [comment]
@martindholmes Yes the problem is from mounting the filesystem, and in nautilus i never saw this. but nemo i also have blocked if i  have an sshfs folder open, and loos the connection to the ssh server. then nemo remains blocked until i force a umount via sudo.

i think nemo has a probelm when a filesystem dont react. maybe nautilus has a timout-logic that nemo is missing. or a better error-handling...

**Comment by SMerrony** [comment]
Still happening in 2026 with a Pixel 9a simply connected...

- Not in developer or debugging mode
- Mint 22.3

**Comment by FibroJedi** [comment]
I am also having this issue on 

- **Device:** Samsung Galaxy S23 
- **OS**: Android
- **USB Port type:** USB-C to USB-C
- Mint 22.2, but if it's Nemo specific that may or may not matter.

To reiterate, my phone is charging only, and I think that's the cable type I'm using, it's not a "data cable".

**Comment by eburger01** [comment]
I'm also having this extremely frustrating issue.  Whenever I plug the phone in, after a while, Files hangs.  As soon as I unplug the phone, Files works again normally.

Phone: Motorola g84 5G
Phone OS: e/OS
Connection: Type C to Type C, Via a hub
Computer: HP EliteBook 845 G8
OS Version: Linux Mint 22.3 - Cinnamon 64-bit
Kernel Version: 7.0.0-30-generic
